package.json

```diff
@@ -44,10 +46,13 @@
     "react-blockies": "1.1.5",
     "react-datetime": "2.8.10",
     "react-dom": "15.6.1",
+    "react-helmet": "5.1.3",
     "react-highcharts": "12.0.0",
     "react-hot-loader": "3.0.0-beta.7",
     "react-modal-dialog": "4.0.7",
     "react-redux": "5.0.5",
+    "react-router-dom": "4.1.1",
+    "react-router-redux": "next",
```

Comment:
RE: that’s actually some cruft left over from an aborted global state approach to path awareness.

Removing...